tests/Integration/Behaviour/behat.yml

```diff
@@ -327,15 +327,30 @@ default:
             filters:
                 tags: "@legacy-product-type"
             contexts:
+                - Tests\Integration\Behaviour\Features\Context\CarrierFeatureContext
```

Comment:
So my suggestion is to create a new suite configuration that mixes context from both Order and Product scopes, but at least it segmentize the different suites

Reply:
I'm mostly suggesting this because I'm worried about the performance cost of loading many contexts in the same suite, and also I'm afraid of the automatic steps BeforeSuite BeforeFeature and such which might come from other contexts and mess with the execution of the whole test